playbooks/os_updates.yml

```diff
@@ -63,20 +63,46 @@
         state: latest
       when: ansible_os_family == "OpenBSD"
 
-    - name: os_updates | Upgrade all packages
+    - name: Ubuntu | Upgrade all packages
       ansible.builtin.apt:
         update_cache: true
         upgrade: dist
         force_apt_get: true
       when: ansible_os_family == "Debian"
 
-    - name: os_updates | Check if reboot required
+    - name: RedHat | Upgrade all packages
+      ansible.builtin.yum:
+        name: "*" 
+        state: latest
+        update_cache: true
+        update_only: true
+      register: yum_update_status
+      when: ansible_os_family == "RedHat"
+
+    - name: RedHat | remove unneeded packages
+      ansible.builtin.yum:
+        autoremove: true
```

Comment:
Oh, interesting, does this work without a package list?

Reply:
If I read this right. https://docs.ansible.com/ansible/latest/collections/ansible/builtin/yum_module.html#parameter-autoremove